cmake/vmmaker.cmake

```diff
@@ -141,8 +141,8 @@ if(GENERATE_SOURCES)
         ExternalProject_Add(
             vmmaker
 
-            URL https://files.pharo.org/image/110/Pharo11-SNAPSHOT.build.688.sha.cf3d3fd.arch.64bit.zip
-            URL_HASH SHA256=c050ddcedce70ec92c22a3244aa5ebbc655dcaffcb42ac80fbf1f6e795c7010d
+            URL https://files.pharo.org/image/110/Pharo11-SNAPSHOT.build.707.sha.f720787.arch.64bit.zip
+            URL_HASH SHA256=b96a943513c6a79c7319b285e46719fbe74c570ace3fa0c9ead43123c165671a
             BUILD_COMMAND ${VMMAKER_VM} --headless ${VMMAKER_DIR}/image/Pharo11-SNAPSHOT-64bit-cf3d3fd.image --no-default-preferences save VMMaker
```

Comment:
Now it is failing because the hash in this line is not right and it does not find the image :)

Reply:
I re-ran ` sha256sum Pharo11-SNAPSHOT.build.707.sha.f720787.arch.64bit.zip` and it gives me that hash that is committed. And the image `Pharo11-SNAPSHOT.build.707.sha.f720787.arch.64bit.zip` is the last one built on files.pharo.org

Reply:
I know, I selected the line below ;)

Reply:
🤦Dumb Sebas